lib/Doctrine/Common/Reflection/StaticReflectionParser.php

```diff
@@ -153,8 +153,10 @@ protected function parse()
                     $docComment = $token[1];
                     break;
                 case T_CLASS:
-                    $this->docComment['class'] = $docComment;
-                    $docComment = '';
+                    if ($docComment) {
```

Comment:
IMO this is more readable, because I don't want to replace docComment with any blank value